src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStoreV47.cs

```diff
@@ -0,0 +1,51 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema.Model;
+using Microsoft.Health.SqlServer.Features.Client;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.Store;
+
+internal class SqlIndexDataStoreV47 : SqlIndexDataStoreV46
+{
+    public SqlIndexDataStoreV47(SqlConnectionWrapperFactory sqlConnectionWrapperFactory)
+        : base(sqlConnectionWrapperFactory)
+    {
+    }
+
+    public override SchemaVersion Version => SchemaVersion.V47;
+
+
+    public override async Task UpdateFrameDataAsync(int partitionKey, IReadOnlyCollection<long> versions, bool hasFrameMetadata, CancellationToken cancellationToken = default)
```

Comment:
nit: could this just be an `IEnumerable<T>` in the input since we don't check the length? Then we don't need to invoke `ToList` as the caller